LibHR/Observables/avr_plaquette.c

```diff
@@ -416,3 +416,182 @@ double complex avr_plaquette_wrk()
 
   return pa;
 }
+
+
+double plaq_APE(int ix,int mu,int nu)
+{
+  int iy,iz;
+  double p;
+  suNg *v1,*v2,*v3,*v4,w1,w2,w3;
+
+  iy=iup(ix,mu);
+  iz=iup(ix,nu);
+
+  v1=pu_gauge_APE(ix,mu);
+  v2=pu_gauge_APE(iy,nu);
+  v3=pu_gauge_APE(iz,mu);
+  v4=pu_gauge_APE(ix,nu);
+
+  _suNg_times_suNg(w1,(*v1),(*v2));
+  _suNg_times_suNg(w2,(*v4),(*v3));
+  _suNg_times_suNg_dagger(w3,w1,w2);
+
+  _suNg_trace_re(p,w3);
+
+#ifdef PLAQ_WEIGHTS
+  if(plaq_weight==NULL) return p;
+  return plaq_weight[ix*16+mu*4+nu]*p;
+#else
+  return p;
+#endif
+}
+
+void cplaq_APE(complex *ret,int ix,int mu,int nu)
+{
+  int iy,iz;
+  suNg *v1,*v2,*v3,*v4,w1,w2,w3;
+  double tmpre = 0.;
+  double tmpim = 0.;
+
+  iy=iup(ix,mu);
+  iz=iup(ix,nu);
+
+  v1=pu_gauge_APE(ix,mu);
+  v2=pu_gauge_APE(iy,nu);
+  v3=pu_gauge_APE(iz,mu);
+  v4=pu_gauge_APE(ix,nu);
+
+  _suNg_times_suNg(w1,(*v1),(*v2));
+  _suNg_times_suNg(w2,(*v4),(*v3));
+  _suNg_times_suNg_dagger(w3,w1,w2);
+
+  _suNg_trace_re(tmpre, w3);
+#ifdef GAUGE_SON
+  _suNg_trace_im(tmpim, w3);
+#else
+  *ret = tmpre + I * tmpim;
+#endif
+
+#ifdef PLAQ_WEIGHTS
+  if(plaq_weight!=NULL) {
+    *ret *= plaq_weight[ix * 16 + mu * 4 + nu];;
+  }
+#endif
+
+}
+
+
+double avr_spacial_plaquette_APE()
```

Comment:
Is there a reason that [`spacial`](https://en.wiktionary.org/wiki/spacial) here is used instead of [`spatial`](https://en.wiktionary.org/wiki/spatial)?